Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -192,7 +192,7 @@ open class SQLiteBookmarksModelFactory: BookmarksModelFactory {
             // The buffer never includes the Places root, so we look one level deeper.
             // Because this is a special-case overlay, we include Mobile Bookmarks here --
             // that'll show bookmarks from other mobile devices.
```

Comment:
It should be removed entirely, no? (this is not longer a 'special case' line of code)